conference.js

```diff
@@ -2343,6 +2345,7 @@ export default {
             'device count: ' + videoDeviceCount);
 
         APP.store.dispatch(setVideoAvailable(available));
+        APP.API.notifyVideoAvailabilityChanged(available);
```

Comment:
I suppose it won't be tricky but personally I would prefer to move the API module first. Anyway it will be another task.